packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -71,5 +71,8 @@
 
   "change_set_based_tools.enabled_description":
     "Specifies whether the commands for change-set-based development are available to the user.",
-  "force_manifest_editor_show_text": "SFDX: Show Package Manifest Editor"
+  "force_manifest_editor_show_text": "SFDX: Show Package Manifest Editor",
+  "force_source_retrieve_text": "SFDX: Retrieve Source from Org",
+  "force_source_retrieve_this_source_file_text":
+    "SFDX: Retrieve This Source File from Org"
```

Comment:
@ruthemmanuelle , this is the text for when a user right clicks in the context of an open file in the editor. The idea is the signal that if they call the command from here, only the current file will be retrieved, but I'm not sure how best to phrase it.
![screen shot 2018-07-17 at 4 13 36 pm](https://user-images.githubusercontent.com/1639375/42848396-91e0872e-89dc-11e8-90fa-c6d15ba8bad8.png)

Reply:
Oh, thanks for the info. Is there a reason this isn't in the i18n.ts file? I usually only look at changes to messages files when PRs come through, so I hadn't seen this change when I approved your PR. This change looks ok though.

Reply:
Thanks for the info, @allileong . If possible I'd prefer that all customer-facing text be in the i18n.ts files. That way if someone decides to translate our extensions into other languages they'll get all of the text, and it also makes my job a lot easier, because whenever pull requests come through I just look at the messages files. (Not everyone always remembers to add me as a reviewer or to point out what has changed.)

Reply:
Ah, interesting. Ok, thanks for trying.